history-of-quakers/modernized/03-chapter-1.adoc

```diff
@@ -1477,7 +1485,14 @@ So when they had wearied themselves in examining him,
 they committed him and another man to the house of correction in Derby, for six months,
 as blasphemers, as appears by the following court order.
 
-To the Master of the House of Correction in Derby, Greetings.
+[.embedded-content-document.legal]
```

Comment:
I may have taken too much liberty in adjusting how this looks. It looks nice in the PDF, but idk if you want it that way!

Reply:
I think it's good! 👍